src/Calculator.js

```diff
@@ -0,0 +1,65 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+class Calculator {
+  async start() {
+    const input = await this.getInput();
+    try {
+      const result = this.calculate(this.parseInt(input));
+      this.printResult(result);
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  async getInput() {
+    const input = await MissionUtils.Console.readLineAsync(
+      "덧셈할 문자열을 입력해 주세요.\n"
+    );
+    return input;
+  }
+
+  parseInt(input) {
+    if (!input) return { delimiters: [",", ":"], numberString: "" };
+
+    let delimiters = [",", ":"];
```

Comment:
```suggestion
    const delimiters = [",", ":"];
```
재할당이 되지 않는 것 같아서 `const` 키워드로 선언해도 무방할 것 같아요~!